lerna.json

```diff
@@ -9,6 +9,8 @@
     "examples/demo-react-native",
     "examples/demo-react-native-detox-instruments",
     "examples/demo-react-native-jest",
+    "examples/demo-puppeteer",
```

Comment:
Where is `demo-puppeteer` project? I cannot find it in the committed files.

Reply:
My mistake. It was a package previously in this PR but it was removed in favor of `demo-plugin`. Fixed in a16ff9b6